xtask/src/main.rs.liquid

```diff
@@ -49,12 +49,14 @@ fn check(subcommand: Option<&str>) {
 
     match subcommand {
         Some("commits") => check_commits(&mut ctx),
+        Some("doc") => check_doc(&mut ctx),
```

Comment:
**suggestion (blocking):** Could we put this after `build`? Either before or after `test`.

Rationale: The first two checks (`commits`, `format`) are low-cost sanity tests. Then, I have ordered the tests by importance order:

1. Build => if it does not build there is a major issue.
2. Lint (part of the build task, using `cargo clippy`) => Helps to see important issue before actually running the code.
3. Test => Run the code to ensure it works properly.
4. Other checks (unused deps, packages, …).

Checking the docs could be seen as another static lint, before the tests, or as a pre-packaging lint, after the tests.

*Don’t forget to reorder in the code below as well.*